internal/pkg/auth/storage_test.go

```diff
@@ -308,8 +318,13 @@ func TestSetGetAuthFieldEncodedTextFile(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.description, func(t *testing.T) {
+			activeProfile, err := config.GetProfile()
+			if err != nil {
+				t.Errorf("get profile: %v", err)
+			}
```

Comment:
For this one we could make the `activeProfile` a field if the test, which would allow us to test these functions for empty profile and active profile cases